draft-ietf-intarea-provisioning-domains.md

```diff
@@ -840,6 +840,58 @@ Note: the above examples assume that the router has received its PvD IDs from up
 or via some other configuration mechanism. Another document could define ways for the router
 to generate its own PvD IDs to allow the above scenario in the absence of PvD ID provisioning.
 
+## Providing Additional Information to PvD-Aware Hosts
+
+In this example, the router indicates that it provides additional information using the H-flag.
+The Sequence Number on the PvD Option is set to 7 in this example.
+
+* RA Header: router lifetime = 6000
+* Prefix Information Option: length = 4, prefix = 2001:db8:cafe::/64
+* Recursive DNS Server Option: length = 3, addresses= \[2001:db8:cafe::53\]
+* PvD Option header: length = 3 + 5 + 4 , PvD ID FQDN = cafe.example.com.,
+Sequence Number = 7, R-flag = 0, H-flag = 1 (actual length of the header with padding
+24 bytes = 3 * 8 bytes)
+	
+A PvD-aware host will fetch https://cafe.example.com/.well-known/pvd to get the additonal
+information. The following example shows a GET request that the host sends:
+
+~~~
+:method = GET
+:scheme = https
+:authority = cafe.example.com
+:path = /.well-known/pvd
+accept = application/pvd+json
+~~~
+
+The HTTP server will respond with the JSON additional information:
+
+~~~
+:status = 200
+content-type = application/pvd+json
+content-length = 116
+
+{
+  "identifier": "cafe.example.com",
+  "expires": "2017-07-23T06:00:00Z",
+  "prefixes": ["2001:db8:cafe::53/48"],
```

Comment:
the prefix should be 2001:db8:cafe::/48 to cover the PIO prefix

Reply:
Fixed!